style.css

```diff
@@ -236,7 +301,20 @@
     padding-top: 16px;
   }
 
-  
+  .button-modal {
+    font-family: Poppins, sans-serif;
+    width: 130px;
+    height: 50px;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    border-color: #396df2;
+    background-color: #fff;
+    margin-left: 10px;
+    border-radius: 10px;
+    margin-top: 10px;
+  }
```

Comment:
- [ ] Good job so far🥇🥇However, while adding the icons, please make sure that they are placed as given in the design guidelines. Kindly add some space between the icon and text and ensure that they are aligned to the center ✨

Your design | Given design
---| ---|
![image](https://github.com/SaveryIV/SaveryIV.github.io/assets/106119105/f1fce6b6-313f-4875-a24e-4af7dee8f989) | ![image](https://github.com/SaveryIV/SaveryIV.github.io/assets/106119105/244085fd-ed51-4efb-801e-30c0659dd7e9)